help/en/package/apps/DecoderPro/DecoderPro.shtml

```diff
@@ -13,68 +10,16 @@
 
 <body>
   <!--#include virtual="/Header.shtml" -->
-  <div class="nomenu" id="mBody">
-    <div id="mainContent">
+  <div id="mBody">
+    <div id="mainContent" class="no-sidebar">
 
       <h1>JMRI: DecoderPro Main Window</h1>
-      <p>The DecoderPro main
-      window gives you access to all the capabilities of JMRI. For
-      more information on DecoderPro itself, please see <a href=
-      "../../../manual/index.shtml">the DecoderPro online
-        manual</a>.</p>
-
-      <p>Help on specific menus:</p>
-
-      <ul>
-        <li>Edit - (<a href=
-        "../TabbedPreferences.shtml">Preferences</a>)</li>
-
-        <li><a href=
-        "../../jmri/jmrit/ToolsMenu.shtml">Tools</a></li>
-
-        <li><a href=
-        "../../../html/apps/DecoderPro/Roster.shtml#menu">Roster</a></li>
-
-        <li><a href=
-        "../../jmri/jmrit/display/PanelMenuHelp.shtml">Panels</a></li>
-
-        <li>(Menus specific to your layout connection will appear
-        here)</li>
-
-        <li><a href=
-        "../../../package/jmri/jmrit/DebugMenu.shtml">Debug</a></li>
-
-        <li>Window - Shows the currently available windows
-
-          <ul>
-            <li>Minimize - minimize the current window, hides
-            it.</li>
-          </ul>
-        </li>
-
-        <li>Help
-
-          <ul>
-            <li>Window Help - Opens a help page specific to the
-            current window. For the main window, that's this
-            page.</li>
-
-            <li>General Help - Opens the main JMRI help page</li>
-
-            <li>License - Displays the JMRI license.</li>
-
-            <li>Locations - Show where the JMRI program and
-            preferences directories are located</li>
 
-            <li>Context - Displays information about your setup of
-            JMRI that might be useful when somebody is trying to
-            help with a problem.</li>
+      <p>The DecoderPro main window gives you access to all the capabilities of JMRI.
+      This is the original DecoderPro that used the same menus as PanelPro.  This version
```

Comment:
I do not understand this comment about "the original DecoderPro", perhaps because I didn't realize there was an "original."

Reply:
Here is the splash screen for the original, which is still available if you know the necessary tricks.

![Screen Shot 2021-07-31 at 10 35 18 PM](https://user-images.githubusercontent.com/17280992/127758331-2cd106bd-39c3-425c-9829-137e5f7caf77.png)

The new DecoderPro was originally called DecoderPro3.  After a couple of years, the original was removed from the JMRI install and DP3 became DP.